### PR TITLE
Proper handling of `SetApplicationCpuTimeLimit` with invalid parameters

### PR DESCRIPTION
**Pull request by noumidev**
`SetApplicationCpuTimeLimit` now returns `Result::OS::NotImplemented` when called with invalid parameters, the CPU time limit is unaffected.
Prior to calling `SetApplicationCpuTimeLimit` for the first time, the CPU time limit returned by `GetApplicationCpuTimeLimit` is 0 on my O3DS, the changes reflect this behavior.

### CODE REVIEW

File: include/result/result_os.hpp [hunk 1]
```diff
@@ -5,10 +5,11 @@ DEFINE_HORIZON_RESULT_MODULE(Result::OS, OS);
 
 namespace Result::OS {
 	DEFINE_HORIZON_RESULT(PortNameTooLong, 30, InvalidArgument, Usage);
-	DEFINE_HORIZON_RESULT(InvalidHandle, 1015, WrongArgument, Permanent);
 	DEFINE_HORIZON_RESULT(InvalidCombination, 1006, InvalidArgument, Usage);
 	DEFINE_HORIZON_RESULT(MisalignedAddress, 1009, InvalidArgument, Usage);
 	DEFINE_HORIZON_RESULT(MisalignedSize, 1010, InvalidArgument, Usage);
+    DEFINE_HORIZON_RESULT(NotImplemented, 1012, InvalidArgument, Usage);
```
Comment (wheremyfoodat) [inline include/result/result_os.hpp:11]: Indentation 😢

Comment (noumidev) [reply include/result/result_os.hpp:11]: No idea how this happened :(

**Review by noumidev — COMMENTED** [review]
indentation is fix, very skill, much prowess

**Comment by wheremyfoodat** [comment]
Much pandess

**Pull request merged**